services/src/util/mod.rs

```diff
@@ -1,12 +1,25 @@
+pub use geoengine_datatypes::util::Identifier;
 use serde::de::Error;
+use serde::{Deserialize, Serialize};
 
-#[macro_use]
-mod identifiers;
-pub use self::identifiers::{IdResponse, Identifier};
 pub mod config;
 mod string_token;
 pub mod user_input;
 
+#[derive(Debug, Serialize, Deserialize, Clone, Copy)]
+pub struct IdResponse<T: Identifier> {
+    pub id: T,
+}
```

Comment:
How is this serialized? Is it just wrapping an id?

Reply:
Yes just a wrapper for a more clean output than just a string

Reply:
Maybe I get this wrong, but an id is an id. Do you want to say
```json
{
  "id": {
    "id": "abc"
  }
}
```
…or is it for working in the code?

Reply:
The `Identifier`s serialize to a simple String containing a UUID. APIs that create something and return the ID like `POST user` would then just output `"9d15d404-dfe7-490a-99ab-d48493368eac"`

Using the wrapper yields
```
{
  "id": "9d15d404-dfe7-490a-99ab-d48493368eac"
}
```
which seems better to me.